### PR TITLE
feat: support to configure the base url of jina.ai

### PR DESCRIPTION
**Pull request by Icemap**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes [#15430](https://github.com/run-llama/llama_index/issues/15430)

## New Package?

Did I fill in the `tool.llamahub` section in the `pyproject.toml` and provide a detailed README.md for my new integration or package?

- [ ] Yes
- [x] No

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [ ] Yes
- [x] No

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

## Suggested Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: llama-index-legacy/llama_index/legacy/embeddings/jinaai.py
Comment (logan-markewich) [inline llama-index-legacy/llama_index/legacy/embeddings/jinaai.py:1]: imo we shouldn't be updating legacy

**Review by logan-markewich — COMMENTED** [review]
Can we bump the toml versions for the package integrations?

File: llama-index-legacy/llama_index/legacy/embeddings/jinaai.py
Comment (Icemap) [reply llama-index-legacy/llama_index/legacy/embeddings/jinaai.py:1]: Sure. Let me change it back.

**Comment by Icemap** [comment]
> Can we bump the toml versions for the package integrations?

Sure, I bumped the toml version from `0.2.0` to `0.2.1`, because it's a small change.

**Review by logan-markewich — APPROVED** [review]

**Pull request merged**